Sources/Preferences/PreferencesStyle.swift

```diff
@@ -1,6 +1,11 @@
 import Cocoa
 
 public enum PreferencesStyle {
+	public enum SegmentSize {
+		case fit
+		case uniform
+	}
+
 	case toolbarItems
-	case segmentedControl
+	case segmentedControl(size: SegmentSize)
```

Comment:
Isn't it technically only the width that changes? Maybe it should be:

```suggestion
	case segmentedControl(width: SegmentWidth)
```
?

Reply:
Although, if we decide to go for shrinking the text too, as mentioned in https://github.com/sindresorhus/Preferences/pull/47#discussion_r382893228, `size` does make sense.